tools/data/transfer.py

```diff
@@ -0,0 +1,96 @@
+"""
+    Moves data from one server to another.
+"""
+import uuid
+import os
+
+from argparse import ArgumentParser
+
+from sshed import servers
+
+
+def parse_input_arguments():
+    arg_parser = ArgumentParser()
+
+    arg_parser.add_argument(
+        '--origin-server', dest='origin_server', required=True,
+        help='Define from which server we need to take the data'
+    )
+
+    arg_parser.add_argument(
+        '--origin-path', dest='origin_path', required=True,
+        help='Define from where we need to take the data'
+    )
+
+    arg_parser.add_argument(
+        '--target-path', dest='target_path', required=True,
+        help='Define where to transfer that data'
+    )
+
+    arg_parser.add_argument(
+        '--target-server', dest='target_server', required=True,
+        help='Define from which server we need to put the data'
+    )
+
+    return arg_parser.parse_args()
+
+
+if __name__ == '__main__':
+    args = parse_input_arguments()
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+
+    task_id = uuid.uuid1()
+
+    print "Task ID: {}".format(str(task_id))
+    print "From: {}".format(args.origin_server)
+    print "Copy: {}".format(args.origin_path)
+    print "To: {}".format(args.target_server)
+    print "Path: {}".format(args.target_path)
+
+    print "\nRead the above input and confirm typing yes"
```

Comment:
can you wrap up by quotes the word "yes" just to highlight what user should type?